active_contracts.json

```diff
@@ -13,23 +13,34 @@
     "PluginRepoRegistry_Implementation": "0x95D563382BeD5AcB458759EE05b27DF2CB019Cc7",
     "PluginRepoRegistry": "0xCe0B4124dea6105bfB85fB4461c4D39f360E9ef3",
     "PluginRepoFactory": "0xE640Da5AD169630555A86D9b6b9C145B4961b1EB",
+    "PluginRepoBase": "0x78744A9664D6A4dDbcd7B73c120535667E739ee3",
     "PluginSetupProcessor": "0x308a1DC5020c4B5d992F5543a7236c465997fecB",
     "DAOFactory": "0x08633901DdF9cD8e2DC3a073594d0A7DaD6f3f57",
+    "DAOBase": "0xbf8b2c4062E1d1060c332bF3305EF03971C90859",
     "AddresslistVotingSetup": "0x7a62da7B56fB3bfCdF70E900787010Bc4c9Ca42e",
+    "AddresslistVotingSetupImplementation": "0x2C9c5e8F559DBBEc962f7CCd295DBc4183cd2168",
     "GovernanceERC20": "0xddCc39a2a0047Eb47EdF94180452cbaB14d426EF",
     "GovernanceWrappedERC20": "0x5B3B36BdC9470963A2734D6a0d2F6a64C21C159f",
     "TokenVotingSetup": "0x96E54098317631641703404C06A5afAD89da7373",
+    "TokenVotingSetupImplementation": "0x4f40F01b2944359A0E507b509a868C8DbFAd215B",
     "AdminSetup": "0xE978942c691e43f65c1B7c7F8f1dc8cDF061B13f",
+    "AdminSetupImplementation": "0xe015E365572c1d778be7E3ACf16f6a0C06dE86f0",
     "MultisigSetup": "0xA03C2182af8eC460D498108C92E8638a580b94d4",
+    "MultisigSetupImplementation": "0x60cD484f4A1B12a652f8ef71D2f55f6d76e64e4A",
     "PlaceholderSetup": "0x6E924eA5864044D8642385683fFA5AD42FB687f2",
+    "PlaceholderSetupImplementation": "0x0000000000000000000000000000000000000000",
     "address-list-voting-repo": "0xf415FF95166EF5D365fFB3bc6d1701f9e9ed7Df7",
+    "address-list-voting-repo_Implementation": "0x78744a9664d6a4ddbcd7b73c120535667e739ee3",
     "token-voting-repo": "0x1AeD2BEb470aeFD65B43f905Bd5371b1E4749d18",
+    "token-voting-repo_Implementation": "0x78744a9664d6a4ddbcd7b73c120535667e739ee3",
     "admin-repo": "0x326A2aee6A8eE78D79E7E956DE60C6E452f76a8e",
-    "multisig-repo": "0x7553E6Fb020c5740768cF289e603770AA09b7aE2"
+    "admin-repo_Implementation": "0x78744a9664d6a4ddbcd7b73c120535667e739ee3",
```

Comment:
can we do `admin-repoImplementation` instead of `admin-repo_Implementation` ?
Same for the rest?